rules_go_gapic/go_gapic.bzl

```diff
@@ -86,6 +86,7 @@ def go_gapic_library(
   gapic_yaml = None,
   samples = [],
   sample_only = False,
+  metadata = False,
```

Comment:
Cool. We'll have to remember to set this to true in our standard publication pipeline. Is there an issue to track this?

Reply:
Good idea. Here is one for `build_gen` https://github.com/googleapis/rules_gapic/issues/10 and another in https://github.com/googleapis/google-cloud-go/issues/3696 (where we have the production pipeline).